packages/analytics/analytics-chart/src/components/chart-types/StackedBarChart.vue

```diff
@@ -359,18 +360,22 @@ const chartWidth = computed(() => {
 })
 
 const chartHeight = computed(() => {
-  let chartHeight = parseInt(props.height, 10)
-  if (numLabels.value && isHorizontal.value) {
+  let value: number | string = DEFAULT_CHART_HEIGHT
+
+  if (canvas.value) {
+    value = canvas.value.width
```

Comment:
suggestion: we already have `{variable}.value` everywhere in Vue; can you rename this `value` variable as something else?

Reply:
done